plugins/postcss-cascade-layers/src/adjust-selector-specificity.ts

```diff
@@ -27,8 +27,8 @@ function generateNot(specificity: number) {
 
 	let list = '';
 	for (let i = 0; i < specificity; i++) {
-		list += ':not(#\\#)'; // something short but still very uncommon
+		list += '#\\#'; // something short but still very uncommon
 	}
 
-	return list;
+	return `:not(${list})`;
```

Comment:
breaks support for older browser versions : https://developer.mozilla.org/en-US/docs/Web/CSS/:not#browser_compatibility